pkg/apis/policy/types.go

```diff
@@ -42,8 +42,37 @@ type PodDisruptionBudgetSpec struct {
 	// by specifying 0. This is a mutually exclusive setting with "minAvailable".
 	// +optional
 	MaxUnavailable *intstr.IntOrString
+
+	// UnreadyPodEvictionPolicy defines the criteria for when unready pods
+	// (no status.conditions item with type=Ready,status=True) should be
+	// considered for eviction. If no policy is specified, the default behavior
+	// will be used, which corresponds to the IfHealthyBudget policy.
+	//
```

Comment:
done, but not sure if the format is correct

also, I think It would be preferable for clients to fail when encountering unrecognized policy. Does this apply to eviction API as well? Should I change the code to fail if policy is unrecognized? 

@liggitt thoughts?

Reply:
I guess clients that just read/display this field just need to be prepared to display additional values.

I agree clients making eviction decisions based on this should default to disallowing eviction of an unready pod if they encounter an unknown policy value. (the only current client making these decisions is the apiserver itself, which will never encounter an unknown value for this field)

how about:
```
	// Additional policies may be added in the future.
  // Clients making eviction decisions should disallow eviction of unready pods if they encounter an unrecognized policy in this field.
```

Reply:
Sounds good - will update the wording. And keep the eviction implementation as is, since we are in control of changing it when we add new policies.